homu/main.py

```diff
@@ -517,10 +517,18 @@ def parse_commands(body, username, repo_cfg, state, my_username, db, states,
                                realtime, my_username):
                 continue
             try:
-                state.priority = int(word[len('p='):])
+                pvalue = int(word[len('p='):])
             except ValueError:
                 pass
 
+            if pvalue > global_cfg['max_priority']:
```

Comment:
Updated to continue there instead. I assume that we do not want to save a non-integer value like "foo" in the state object, as it might pop out and break things when it's read elsewhere.